scripts/restore-sanitised-database.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+clean(){
+    echo "Removing sensitive values"
+    unset unset DB_TO_RESTORE_PASSWORD
+    kill $SSM_PID
+}
+
+# Function to list and select an AWS Vault profile
+select_aws_vault_profile() {
+    local prompt_message=$1
+    local profile_variable=$2
+
+    echo "$prompt_message"
+    PS3='Please select a profile (or Ctrl-C to quit): '
+    IFS=$'\n' read -r -d '' -a profiles <<< "$(aws-vault list --profiles)"
+    select profile in "${profiles[@]}"; do
+        if [[ -n "$profile" ]]; then
+            echo "Selected profile: $profile"
+            export "$profile_variable"="$profile"
+            break
+        else
+            echo "Invalid selection, please try again."
+        fi
+    done
+}
+
+# Prompt user for AWS profiles based on environment
+select_aws_vault_profile "Select the AWS Vault profile for the ${ENV} environment:" AWS_VAULT_PROFILE_TO_SANITISE
+echo "You will use the ${AWS_VAULT_PROFILE_TO_SANITISE} aws-vault profile. Ctrl-C now if this is wrong..."
+
+# Retrieve AWS account ID using the selected profile
+ACCOUNT=$(aws-vault exec $AWS_VAULT_PROFILE_TO_SANITISE -- aws sts get-caller-identity | yq -r ".Account")
+echo "Connecting through account ${ACCOUNT}"
+
+# Determine environment based on account ID
+case $ACCOUNT in
+    012*)echo "(dev)"ADD=0;;
+    960*)echo "(test)"ADD=10;;
+    233*)echo "(prod)"ADD=30
+        echo -e "==========================="
+        echo "Running in production is not allowed. Please select 'dev' or 'test' instead."
+        # Prompt the user to select a different environment
+        while true; do
+            echo "Please select a different environment (dev or test):"
+            read -p "Enter 'dev' or 'test': " ENV
+
+            if [[ "$ENV" == "dev" ]]; then
+                echo "You selected dev environment."
+                ADD=0
+                break
+            elif [[ "$ENV" == "test" ]]; then
+                echo "You selected test environment."
+                ADD=10
+                break
+            else
+                echo "Invalid selection. Please choose 'dev' or 'test'."
+            fi
+        done
+        ;;
+    *)
+        echo "INVALID ACCOUNT!"
+        exit 1
+        ;;
+esac
+
+# Variables based on the environment
+sanitised-db="Sanitised-database.sql"
+S3_BUCKET="aws-s3-bucket-sanitise-database"
```

Comment:
Sorry if I'm missing something but if the bucket is created by copilot it will have more like a random unique value in the name, so would we not need a way to specify that as an argument to the script?

Reply:
This bucket is created manually and nothing done using copilot atm, sorry if we missed that. Also we still don't have any bucket for the anon stuff in prod or test so this only exists in dev.

Reply:
To make this production ready, I think we should make this configurable/overridable.